uwblueprint/project-read-backend
# add options to Field model

add options to Field model and modify tests under registration

### Notion ticket link



[Add options for MC questions](https://www.notion.so/uwblueprintexecs/Add-options-for-multiple-choice-questions-eb08b5beafaa4257b94344b661703c40)



### Implementation description

- added options as array of strings
- added options to serializer
- updated dummy data scripts and tests as necessary to use values from MC options



### Steps to test

1. python manage.py test
2. go to Django Admin and click on a student to make sure all field values make sense



### What should reviewers focus on?

- functionality
- code/test quality

### Checklist
